libs/use-viewer/src/utils.ts

```diff
@@ -67,17 +72,70 @@ export const buildQualityOptions = (layers: MediaLayer[]) => {
     },
     streamQuality: qualities[idx],
   }));
-  return qualityOptions;
+
+  return [{ streamQuality: 'Auto' } as SimulcastQuality, ...qualityOptions];
+};
+
+export const reprojectToMapping = async (
```

Comment:
The name is not good, we project the video and/or audio tracks in one source to a MediaStream which is represented by the `mids` in mapping.  So we could call it `projectToStream`